App crashes if playback fails

### Describe the bug

1. Attempt to play media
   1. In this instance I played LiveTV
1. Have media playback fail
1. App crashes 

I don't know if this is new to 0.18.0-beta.1 

### Logs

---
client: Jellyfin for Android TV
client_version: 0.18.0-beta.1
client_repository: https://github.com/jellyfin/jellyfin-androidtv
type: crash_report
format: markdown
---
### Logs

***Stack Trace***: 
```log
java.lang.NullPointerException: Parameter specified as non-null is null: method org.jellyfin.androidtv.util.apiclient.ReportingHelper.reportStopped, parameter streamInfo
	at org.jellyfin.androidtv.util.apiclient.ReportingHelper.reportStopped(Unknown Source:20)
	at org.jellyfin.androidtv.ui.playback.PlaybackController.stop(PlaybackController.java:779)
	at org.jellyfin.androidtv.ui.playback.PlaybackController.endPlayback(PlaybackController.java:794)
	at org.jellyfin.androidtv.ui.playback.PlaybackController.endPlayback(PlaybackController.java:803)
	at org.jellyfin.androidtv.ui.playback.CustomPlaybackOverlayFragment.onStop(CustomPlaybackOverlayFragment.java:698)
	at androidx.fragment.app.Fragment.performStop(Fragment.java:3346)
	at androidx.fragment.app.FragmentStateManager.stop(FragmentStateManager.java:700)
	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:324)
	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2214)
	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:2115)
	at androidx.fragment.app.FragmentManager.execSingleAction(FragmentManager.java:2002)
	at androidx.fragment.app.BackStackRecord.commitNow(BackStackRecord.java:317)
	at org.jellyfin.androidtv.ui.browsing.DestinationFragmentView.activateHistoryEntry(DestinationFragmentView.kt:159)
	at org.jellyfin.androidtv.ui.browsing.DestinationFragmentView.goBack(DestinationFragmentView.kt:113)
	at org.jellyfin.androidtv.ui.browsing.MainActivity.handleNavigationAction(MainActivity.kt:117)
	at org.jellyfin.androidtv.ui.browsing.MainActivity.access$handleNavigationAction(MainActivity.kt:33)
	at org.jellyfin.androidtv.ui.browsing.MainActivity$onCreate$2.invokeSuspend(MainActivity.kt:66)
	at org.jellyfin.androidtv.ui.browsing.MainActivity$onCreate$2.invoke(Unknown Source:8)
	at org.jellyfin.androidtv.ui.browsing.MainActivity$onCreate$2.invoke(Unknown Source:4)
	at kotlinx.coroutines.flow.FlowKt__TransformKt$onEach$$inlined$unsafeTransform$1$2.emit(Emitters.kt:50)
	at kotlinx.coroutines.flow.FlowKt__ChannelsKt.emitAllImpl$FlowKt__ChannelsKt(Channels.kt:33)
	at kotlinx.coroutines.flow.FlowKt__ChannelsKt.access$emitAllImpl$FlowKt__ChannelsKt(Channels.kt:1)
	at kotlinx.coroutines.flow.FlowKt__ChannelsKt$emitAllImpl$1.invokeSuspend(Unknown Source:14)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:101)
	at kotlinx.coroutines.EventLoop.processUnconfinedEvent(EventLoop.common.kt:65)
	at kotlinx.coroutines.DispatchedTaskKt.resumeUnconfined(DispatchedTask.kt:231)
	at kotlinx.coroutines.DispatchedTaskKt.dispatch(DispatchedTask.kt:149)
	at kotlinx.coroutines.CancellableContinuationImpl.dispatchResume(CancellableContinuationImpl.kt:470)
	at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$kotlinx_coroutines_core(CancellableContinuationImpl.kt:504)
	at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$kotlinx_coroutines_core$default(CancellableContinuationImpl.kt:493)
	at kotlinx.coroutines.CancellableContinuationImpl.resumeWith(CancellableContinuationImpl.kt:359)
	at kotlinx.coroutines.flow.SharedFlowImpl.tryEmit(SharedFlow.kt:414)
	at org.jellyfin.androidtv.ui.navigation.NavigationRepositoryImpl.goBack(NavigationRepository.kt:93)
	at org.jellyfin.androidtv.ui.playback.CustomPlaybackOverlayFragment.closePlayer(CustomPlaybackOverlayFragment.java:1298)
	at org.jellyfin.androidtv.ui.playback.PlaybackController.handlePlaybackInfoError(PlaybackController.java:578)
	at org.jellyfin.androidtv.ui.playback.PlaybackController.access$200(PlaybackController.java:54)
	at org.jellyfin.androidtv.ui.playback.PlaybackController$4.onError(PlaybackController.java:534)
	at org.jellyfin.apiclient.interaction.Response.onError(Response.java:32)
	at org.jellyfin.apiclient.interaction.Response.onError(Response.java:32)
	at org.jellyfin.apiclient.interaction.Response.onError(Response.java:32)
	at org.jellyfin.apiclient.interaction.ApiClientRequestListener.onError(ApiClientRequestListener.java:37)
	at org.jellyfin.apiclient.interaction.VolleyErrorListener.onErrorResponse(VolleyErrorListener.java:36)
	at com.android.volley.Request.deliverError(Request.java:648)
	at com.android.volley.ExecutorDelivery$ResponseDeliveryRunnable.run(ExecutorDelivery.java:104)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7656)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@a1a9d94, Dispatchers.Main.immediate]

```  
***Logcat***: 
```log
11-03 08:43:02.680 D/org.jellyfin.sdk.api.sockets.OkHttpSocketConnection( 7415): Receiving (raw) message {"MessageId":"174c4dc322d44424abd1b3ef6b6ffe62","MessageType":"KeepAlive"}
11-03 08:43:02.751 I/PlaybackController( 7415): Play called from state: BUFFERING with pos: 0 and sub index: null
11-03 08:43:02.772 D/org.jellyfin.sdk.api.sockets.OkHttpSocketConnection( 7415): Receiving (raw) message {"MessageId":"48b3e1eb8d534cd0abab30d05f8b8d01","Data":{"ItemId":"c6004915616ad758f5acb31261971138","Progress":"59.024752475247524"},"MessageType":"RefreshProgress"}
11-03 08:43:02.773 D/org.jellyfin.sdk.api.sockets.OkHttpSocketConnection( 7415): Receiving (raw) message {"MessageId":"fce06cfbe18540e4a8af02c1a91d5d13","Data":{"ItemId":"7e64e319657a9516ec78490da03edccb","Progress":"59.024752475247524"},"MessageType":"RefreshProgress"}
11-03 08:43:08.438 D/org.jellyfin.sdk.api.sockets.OkHttpSocketConnection( 7415): Receiving (raw) message {"MessageId":"9a5684c916f2431fb1a87600998605b3","Data":{"ItemId":"c6004915616ad758f5acb31261971138","Progress":"59.24752475247524"},"MessageType":"RefreshProgress"}
11-03 08:43:08.439 D/org.jellyfin.sdk.api.sockets.OkHttpSocketConnection( 7415): Receiving (raw) message {"MessageId":"a0f83a97f4da4996b088070360c18772","Data":{"ItemId":"7e64e319657a9516ec78490da03edccb","Progress":"59.24752475247524"},"MessageType":"RefreshProgress"}
11-03 08:43:08.445 E/jellyfin-apiclient( 7415): VolleyError com.android.volley.TimeoutError: null
11-03 08:43:08.445 E/jellyfin-apiclient( 7415): com.android.volley.TimeoutError
11-03 08:43:08.445 E/jellyfin-apiclient( 7415): 	at com.android.volley.toolbox.NetworkUtility.shouldRetryException(NetworkUtility.java:162)
11-03 08:43:08.445 E/jellyfin-apiclient( 7415): 	at com.android.volley.toolbox.BasicNetwork.performRequest(BasicNetwork.java:145)
11-03 08:43:08.445 E/jellyfin-apiclient( 7415): 	at com.android.volley.NetworkDispatcher.processRequest(NetworkDispatcher.java:132)
11-03 08:43:08.445 E/jellyfin-apiclient( 7415): 	at com.android.volley.NetworkDispatcher.processRequest(NetworkDispatcher.java:111)
11-03 08:43:08.445 E/jellyfin-apiclient( 7415): 	at com.android.volley.NetworkDispatcher.run(NetworkDispatcher.java:90)
11-03 08:43:08.450 E/PlaybackController( 7415): Error getting playback stream info
11-03 08:43:08.450 E/PlaybackController( 7415): org.jellyfin.apiclient.model.net.HttpException: VolleyError com.android.volley.TimeoutError: 
11-03 08:43:08.450 E/PlaybackController( 7415): 	at org.jellyfin.apiclient.interaction.VolleyErrorListener.onErrorResponse(VolleyErrorListener.java:25)
11-03 08:43:08.450 E/PlaybackController( 7415): 	at com.android.volley.Request.deliverError(Request.java:648)
11-03 08:43:08.450 E/PlaybackController( 7415): 	at com.android.volley.ExecutorDelivery$ResponseDeliveryRunnable.run(ExecutorDelivery.java:104)
11-03 08:43:08.450 E/PlaybackController( 7415): 	at android.os.Handler.handleCallback(Handler.java:938)
11-03 08:43:08.450 E/PlaybackController( 7415): 	at android.os.Handler.dispatchMessage(Handler.java:99)
11-03 08:43:08.450 E/PlaybackController( 7415): 	at android.os.Looper.loop(Looper.java:223)
11-03 08:43:08.450 E/PlaybackController( 7415): 	at android.app.ActivityThread.main(ActivityThread.java:7656)
11-03 08:43:08.450 E/PlaybackController( 7415): 	at java.lang.reflect.Method.invoke(Native Method)
11-03 08:43:08.450 E/PlaybackController( 7415): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
11-03 08:43:08.450 E/PlaybackController( 7415): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
11-03 08:43:08.450 E/PlaybackController( 7415): Caused by: com.android.volley.TimeoutError
11-03 08:43:08.450 E/PlaybackController( 7415): 	at com.android.volley.toolbox.NetworkUtility.shouldRetryException(NetworkUtility.java:162)
11-03 08:43:08.450 E/PlaybackController( 7415): 	at com.android.volley.toolbox.BasicNetwork.performRequest(BasicNetwork.java:145)
11-03 08:43:08.450 E/PlaybackController( 7415): 	at com.android.volley.NetworkDispatcher.processRequest(NetworkDispatcher.java:132)
11-03 08:43:08.450 E/PlaybackController( 7415): 	at com.android.volley.NetworkDispatcher.processRequest(NetworkDispatcher.java:111)
11-03 08:43:08.450 E/PlaybackController( 7415): 	at com.android.volley.NetworkDispatcher.run(NetworkDispatcher.java:90)
11-03 08:43:08.452 D/CompatibilityChangeReporter( 7415): Compat change id reported: 147798919; UID 10096; state: ENABLED
11-03 08:43:08.476 D/NavigationRepositoryImpl( 7415): Navigating back
11-03 08:43:08.487 D/CustomPlaybackOverlayFragment( 7415): Stopping!
11-03 08:43:08.488 D/CustomPlaybackOverlayFragment( 7415): this fragment belongs to the current session, ending it
11-03 08:43:08.489 D/PlaybackController( 7415): stop called at 0
11-03 08:43:08.505 E/ACRA    ( 7415): ACRA caught a NullPointerException for org.jellyfin.androidtv.debug
11-03 08:43:08.505 E/ACRA    ( 7415): java.lang.NullPointerException: Parameter specified as non-null is null: method org.jellyfin.androidtv.util.apiclient.ReportingHelper.reportStopped, parameter streamInfo
11-03 08:43:08.505 E/ACRA    ( 7415): 	at org.jellyfin.androidtv.util.apiclient.ReportingHelper.reportStopped(Unknown Source:20)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at org.jellyfin.androidtv.ui.playback.PlaybackController.stop(PlaybackController.java:779)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at org.jellyfin.androidtv.ui.playback.PlaybackController.endPlayback(PlaybackController.java:794)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at org.jellyfin.androidtv.ui.playback.PlaybackController.endPlayback(PlaybackController.java:803)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at org.jellyfin.androidtv.ui.playback.CustomPlaybackOverlayFragment.onStop(CustomPlaybackOverlayFragment.java:698)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at androidx.fragment.app.Fragment.performStop(Fragment.java:3346)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at androidx.fragment.app.FragmentStateManager.stop(FragmentStateManager.java:700)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:324)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2214)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:2115)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at androidx.fragment.app.FragmentManager.execSingleAction(FragmentManager.java:2002)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at androidx.fragment.app.BackStackRecord.commitNow(BackStackRecord.java:317)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at org.jellyfin.androidtv.ui.browsing.DestinationFragmentView.activateHistoryEntry(DestinationFragmentView.kt:159)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at org.jellyfin.androidtv.ui.browsing.DestinationFragmentView.goBack(DestinationFragmentView.kt:113)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at org.jellyfin.androidtv.ui.browsing.MainActivity.handleNavigationAction(MainActivity.kt:117)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at org.jellyfin.androidtv.ui.browsing.MainActivity.access$handleNavigationAction(MainActivity.kt:33)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at org.jellyfin.androidtv.ui.browsing.MainActivity$onCreate$2.invokeSuspend(MainActivity.kt:66)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at org.jellyfin.androidtv.ui.browsing.MainActivity$onCreate$2.invoke(Unknown Source:8)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at org.jellyfin.androidtv.ui.browsing.MainActivity$onCreate$2.invoke(Unknown Source:4)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at kotlinx.coroutines.flow.FlowKt__TransformKt$onEach$$inlined$unsafeTransform$1$2.emit(Emitters.kt:50)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at kotlinx.coroutines.flow.FlowKt__ChannelsKt.emitAllImpl$FlowKt__ChannelsKt(Channels.kt:33)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at kotlinx.coroutines.flow.FlowKt__ChannelsKt.access$emitAllImpl$FlowKt__ChannelsKt(Channels.kt:1)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at kotlinx.coroutines.flow.FlowKt__ChannelsKt$emitAllImpl$1.invokeSuspend(Unknown Source:14)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:101)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at kotlinx.coroutines.EventLoop.processUnconfinedEvent(EventLoop.common.kt:65)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at kotlinx.coroutines.DispatchedTaskKt.resumeUnconfined(DispatchedTask.kt:231)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at kotlinx.coroutines.DispatchedTaskKt.dispatch(DispatchedTask.kt:149)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at kotlinx.coroutines.CancellableContinuationImpl.dispatchResume(CancellableContinuationImpl.kt:470)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$kotlinx_coroutines_core(CancellableContinuationImpl.kt:504)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at kotlinx.coroutines.CancellableContinuationImpl.resumeImpl$kotlinx_coroutines_core$default(CancellableContinuationImpl.kt:493)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at kotlinx.coroutines.CancellableContinuationImpl.resumeWith(CancellableContinuationImpl.kt:359)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at kotlinx.coroutines.flow.SharedFlowImpl.tryEmit(SharedFlow.kt:414)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at org.jellyfin.androidtv.ui.navigation.NavigationRepositoryImpl.goBack(NavigationRepository.kt:93)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at org.jellyfin.androidtv.ui.playback.CustomPlaybackOverlayFragment.closePlayer(CustomPlaybackOverlayFragment.java:1298)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at org.jellyfin.androidtv.ui.playback.PlaybackController.handlePlaybackInfoError(PlaybackController.java:578)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at org.jellyfin.androidtv.ui.playback.PlaybackController.access$200(PlaybackController.java:54)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at org.jellyfin.androidtv.ui.playback.PlaybackController$4.onError(PlaybackController.java:534)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at org.jellyfin.apiclient.interaction.Response.onError(Response.java:32)
11-03 08:43:08.505 E/ACRA    ( 7415): 	at org.jellyfin.apiclient.interaction.Response.onError(Response.java:32)
11-03 08:43:08.507 E/ACRA    ( 7415): 	at org.jellyfin.apiclient.interaction.Response.onError(Response.java:32)
11-03 08:43:08.507 E/ACRA    ( 7415): 	at org.jellyfin.apiclient.interaction.ApiClientRequestListener.onError(ApiClientRequestListener.java:37)
11-03 08:43:08.507 E/ACRA    ( 7415): 	at org.jellyfin.apiclient.interaction.VolleyErrorListener.onErrorResponse(VolleyErrorListener.java:36)
11-03 08:43:08.507 E/ACRA    ( 7415): 	at com.android.volley.Request.deliverError(Request.java:648)
11-03 08:43:08.507 E/ACRA    ( 7415): 	at com.android.volley.ExecutorDelivery$ResponseDeliveryRunnable.run(ExecutorDelivery.java:104)
11-03 08:43:08.507 E/ACRA    ( 7415): 	at android.os.Handler.handleCallback(Handler.java:938)
11-03 08:43:08.507 E/ACRA    ( 7415): 	at android.os.Handler.dispatchMessage(Handler.java:99)
11-03 08:43:08.507 E/ACRA    ( 7415): 	at android.os.Looper.loop(Looper.java:223)
11-03 08:43:08.507 E/ACRA    ( 7415): 	at android.app.ActivityThread.main(ActivityThread.java:7656)
11-03 08:43:08.507 E/ACRA    ( 7415): 	at java.lang.reflect.Method.invoke(Native Method)
11-03 08:43:08.507 E/ACRA    ( 7415): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
11-03 08:43:08.507 E/ACRA    ( 7415): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
11-03 08:43:08.507 E/ACRA    ( 7415): 	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@a1a9d94, Dispatchers.Main.immediate]
11-03 08:43:08.507 D/ACRA    ( 7415): Building report
11-03 08:43:08.510 D/ACRA    ( 7415): Starting collectors with priority FIRST
11-03 08:43:08.511 D/ACRA    ( 7415): Calling collector org.acra.collector.LogCatCollector
11-03 08:43:08.511 D/ACRA    ( 7415): Calling collector org.acra.collector.DropBoxCollector
11-03 08:43:08.512 D/ACRA    ( 7415): Collector org.acra.collector.DropBoxCollector completed
11-03 08:43:08.513 D/ACRA    ( 7415): Calling collector org.acra.collector.StacktraceCollector
11-03 08:43:08.515 D/ACRA    ( 7415): Collector org.acra.collector.StacktraceCollector completed
11-03 08:43:08.522 D/ACRA    ( 7415): Retrieving logcat output (buffer:default)...

```  

### App information

***App version***: `0.18.0-beta.1` (`180001`)  
***Package name***: `org.jellyfin.androidtv.debug`  
***Build***: 
```json
{"BOARD":"YYJ","BOOTLOADER":"01.01.230509.145718","BRAND":"Homatics","CPU_ABI":"armeabi-v7a","CPU_ABI2":"armeabi","DEVICE":"YYJ","DISPLAY":"RTT0.211009.001.6261 release-keys","FINGERPRINT":"Homatics\/SEI804HM\/YYJ:11\/RTT0.211009.001\/6261:user\/release-keys","HARDWARE":"amlogic","HOST":"sei-Precision-3640-Tower","ID":"RTT0.211009.001","IS_DEBUGGABLE":false,"IS_EMULATOR":false,"MANUFACTURER":"SEI Robotics","MODEL":"Box R 4K Plus","PERMISSIONS_REVIEW_REQUIRED":true,"PRODUCT":"SEI804HM","RADIO":"unknown","SUPPORTED_32_BIT_ABIS":["armeabi-v7a","armeabi"],"SUPPORTED_64_BIT_ABIS":[],"SUPPORTED_ABIS":["armeabi-v7a","armeabi"],"TAGS":"release-keys","TIME":1701239516000,"TYPE":"user","UNKNOWN":"unknown","USER":"jenkins","VERSION":{"ACTIVE_CODENAMES":[],"BASE_OS":"","CODENAME":"REL","INCREMENTAL":"6261","PREVIEW_SDK_FINGERPRINT":"REL","PREVIEW_SDK_INT":0,"RELEASE":"11","RELEASE_OR_CODENAME":"11","SDK":"30","SDK_INT":30,"SECURITY_PATCH":"2023-10-05"}}
```  
***Build config***: 
```json
{"APPLICATION_ID":"org.jellyfin.androidtv.debug","BUILD_TYPE":"debug","DEBUG":true,"DEVELOPMENT":false,"VERSION_CODE":180001,"VERSION_NAME":"0.18.0-beta.1"}
```  

### Device information

***Android version***: `11`  
***Device brand***: `Homatics`  
***Device product***: `SEI804HM`  
***Device model***: `Box R 4K Plus`  

### Crash information

***Start time***: `2024-11-03T08:19:58.428-07:00`  
***Crash time***: `2024-11-03T08:43:08.624-07:00`  



### Application version

0.18.0-beta.1

### Where did you install the app from?

Sideloaded APK

### Device information

Box R 4K Plus

### Android version

Android 11

### Jellyfin server version

10.10.0

--- a/app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java
+++ b/app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java
@@ -775,8 +775,10 @@ public class PlaybackController implements PlaybackControllerNotifiable {
             mPlaybackState = PlaybackState.IDLE;
 
             if (mVideoManager != null && mVideoManager.isPlaying()) mVideoManager.stopPlayback();
-            Long mbPos = mCurrentPosition * 10000;
-            reportingHelper.getValue().reportStopped(mFragment, getCurrentlyPlayingItem(), getCurrentStreamInfo(), mbPos);
+            if (getCurrentlyPlayingItem() != null && mCurrentStreamInfo != null) {
+                Long mbPos = mCurrentPosition * 10000;
+                reportingHelper.getValue().reportStopped(mFragment, getCurrentlyPlayingItem(), mCurrentStreamInfo, mbPos);
+            }
             clearPlaybackSessionOptions();
         }
     }

